backend/app/quiz_creation/conjugation_quiz.py

```diff
@@ -0,0 +1,189 @@
+import random
+from app.analysis.parts_of_speech import get_parts_of_speech_tags, CONTRACTIONS
+from app.quiz_creation.verb_conjugation import (
+    verb_tenses,
+    verb_tense,
+    verb_tenses_keys,
+    verb_infinitive,
+)
+
+
+def get_sentence_options(word):
+    """
+    Creates the answer choices for a sentence in the quiz game
+    :param word: A verb (string) that is the answer to the question
+    :return: A list of 4 words to serve as the answer choices
+    """
+    options = [word]
+    word_is_capitalized = word[0].isupper()
+    # Check if tense is found in the tenses dictionary
+    tenses = verb_tenses.get(verb_infinitive(word), '')
+    if tenses:
+        # Determine the number of discrete tenses of the verb
+        discrete_tenses = []
+        for tense in tenses:
+            if tense != '' and tense != word and tense not in discrete_tenses:
+                discrete_tenses.append(tense)
+
+        if len(discrete_tenses) >= 3:
+            options += random.sample(discrete_tenses, 3)
+        else:
+            # If the verb does not have at least 3 discrete verb tenses, add the same tense of
+            # another verb. ex: word = walked, other_options = baked, raked, etc.
+            verb_form = verb_tense(word)
+            key = verb_tenses_keys[verb_form]
+            verbs_of_same_tense = [
+                verb_tenses[verb][key] for verb in verb_tenses if verb_tenses[verb][key] != ''
+            ]
+            options += discrete_tenses
+            options += random.sample(verbs_of_same_tense, 3 - len(discrete_tenses))
+    else:
+        # If the verb is not in the verb_tenses dictionary, get three random verb infinitives
+        # (keys) from the dictionary
+        options += random.sample(list(verb_tenses), 3)
+
+    # Capitalize other options if the original word was capitalized
+    if word_is_capitalized:
+        for i in range(4):
+            options[i] = options[i].capitalize()
+
+    random.shuffle(options)
+
+    # Capitalize the options if verb is at the beginning of the sentence
+    if word[0].isupper():
+        return [w.capitalize() for w in options]
+
+    return options
+
+
+def process_text(text):
```

Comment:
This name is vague ("process" could mean any data transformation at all). I think this work might be a bit redundant with some work over in `app.analysis.parts_of_speech` -- let's do a cleanup refactor so that all of this text cleanup stuff is in one place.